aries-site/src/examples/components/notifications/TaxonomyTable.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { Box, DataTable, Text } from 'grommet';
+
+const data = [
+    {
+        type: 'Information',
+        state: 'In-Progress',
+        status: 'Critical',
+        attentionLevel: 'High',
+        actions: 'Dismiss',
+        persistence: 'Dismissable',
+        placement: 'Banner',
+        insight: 'Support',
+    },
+    {
+        type: 'Task',
+        state: 'Starting',
+        status: 'Normal',
+        attentionLevel: 'Medium',
+        actions: 'Read',
+        persistence: 'Clearable',
+        placement: 'Browser',
+        insight: 'Progress Update',
+    },
+    {
+        type: 'Event',
+        state: 'Stopped',
+        status: 'Warning',
+        attentionLevel: 'Low',
+        actions: 'Unread',
+        persistence: 'Action Required',
+        placement: 'Email',
+        insight: 'Status/Task Tracking',
+    },
+    {
+        type: '',
+        state: 'Pending',
+        status: 'Unknown',
+        attentionLevel: '',
+        actions: 'Yes',
+        persistence: 'Temporary',
+        placement: 'Toast',
+        insight: 'Created',
+    },
+    {
+        type: '',
+        state: 'Success',
+        status: '',
+        attentionLevel: '',
+        actions: 'No',
+        persistence: '',
+        placement: 'Bell',
+        insight: 'Request Received',
+    },
+    {
+        type: '',
+        state: 'Failure',
+        status: '',
+        attentionLevel: '',
+        actions: '',
+        persistence: '',
+        placement: 'Badging',
+        insight: '',
+    },
+    {
+        type: '',
+        state: '',
+        status: '',
+        attentionLevel: '',
+        actions: '',
+        persistence: '',
+        placement: 'System',
+        insight: '',
+    },
+    {
+        type: '',
+        state: '',
+        status: '',
+        attentionLevel: '',
+        actions: '',
+        persistence: '',
+        placement: 'Global',
+        insight: '',
+    },
+    {
+        type: '',
+        state: '',
+        status: '',
+        attentionLevel: '',
+        actions: '',
+        persistence: '',
+        placement: 'Inline',
+        insight: '',
+    },
+];
+
+export const columns = [
+    {
+        property: 'type',
+        header: <Text weight="bold">Type</Text>,
```

Comment:
```suggestion
        header: <Text color="text-strong" weight="bold">Type</Text>,
```
Can we add color="text-strong" to all these table headings to align with way we style datatable headings?